presto-thrift-testing-server/src/main/java/com/facebook/presto/connector/thrift/server/ColumnWriters.java

```diff
@@ -0,0 +1,237 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.thrift.server;
+
+import com.facebook.presto.connector.thrift.api.PrestoThriftBlock;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftBigint;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftBoolean;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftDate;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftDouble;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftHyperLogLog;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftInteger;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftJson;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftTimestamp;
+import com.facebook.presto.connector.thrift.api.datatypes.PrestoThriftVarchar;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.type.Type;
+import io.airlift.slice.Slice;
+
+import java.util.function.BiFunction;
+
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.bigintData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.booleanData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.dateData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.doubleData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.hyperLogLogData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.integerData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.jsonData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.timestampData;
+import static com.facebook.presto.connector.thrift.api.PrestoThriftBlock.varcharData;
+import static com.facebook.presto.spi.type.StandardTypes.BIGINT;
+import static com.facebook.presto.spi.type.StandardTypes.BOOLEAN;
+import static com.facebook.presto.spi.type.StandardTypes.DATE;
+import static com.facebook.presto.spi.type.StandardTypes.DOUBLE;
+import static com.facebook.presto.spi.type.StandardTypes.HYPER_LOG_LOG;
+import static com.facebook.presto.spi.type.StandardTypes.INTEGER;
+import static com.facebook.presto.spi.type.StandardTypes.JSON;
+import static com.facebook.presto.spi.type.StandardTypes.TIMESTAMP;
+import static com.facebook.presto.spi.type.StandardTypes.VARCHAR;
+import static com.google.common.base.Preconditions.checkState;
+
+public final class ColumnWriters
```

Comment:
This doesn't seem like the correct name anymore... maybe `PrestoThriftBlockFactory`... what ever you change this to, change the test name also.